types/time.go

```diff
@@ -1219,13 +1615,29 @@ func ParseTimeFromFloatString(sc *stmtctx.StatementContext, str string, tp byte,
 	return parseTime(sc, str, tp, fsp, true)
 }
 
+func parseTimeNoFsp(sc *stmtctx.StatementContext, str string, tp byte, isFloat bool) (Time, error) {
+
+	var fsp = MinFsp
+	var selfDecideFsp = true
+	t, err := parseDatetimeV2(str, fsp, isFloat, selfDecideFsp)
```

Comment:
parseDatetimeV2(str, fsp, isFloat, `true`)
There is no need to define a variable `selfDecideFsp` here.

Reply:
ok

Reply:
For improving readability. How do u think?